tests/php/media/test-class.jetpack-media-extractor.php

```diff
@@ -377,7 +377,7 @@ function test_extract_images() {
 	/**
 	 * @author scotchfield
 	 * @covers Jetpack_Media_Meta_Extractor::extract
-	 * @todo This test is failing in 5.2 and 5.3 for unknown reasons. Figure it out.
+	 * @todo This test is failing in 5.3 for unknown reasons. Figure it out.
```

Comment:
I'm not sure, but do we still support PHP 5.3?

Reply:
Right now we do: https://github.com/Automattic/jetpack/blob/5a2bb2b2423a42a59ced3674f30339605fbc7fd5/jetpack.php#L17

I'd expect we'll continue supporting it at least until the next major WP version is released.

Reply:
not anymore

Reply:
We still support 5.3 officially, but run tests only on 5.6 and above. Do you think we should drop that too in this case?

Reply:
not anymore :) https://github.com/Automattic/jetpack/pull/12941 

but I would ask @dereksmart first, they are considering bringing 5.3 support back